ilastik/utility/slot_name_enum.py

```diff
@@ -0,0 +1,50 @@
+import enum
+from typing import List, Tuple
+from itertools import chain
+
+class SlotNameEnum(enum.IntEnum):
+    """A map from slot names to their indices within a multislot
+
+    Do note that the ENUM_NAMEs that you pick matter, as they determine
+    the human readable version that you get from instances of this enum:
+
+    e.g a value defined like so:
+        MY_DATA_SLOT = 123
+    will have its slotName rendered as
+        'My Data Slot'
+    """
+
+    @property
+    def slotName(self) -> str:
```

Comment:
This enum is just replicating some logic already in ilastik, and that goes for the name generation and command line flag names. Really, I don't even think this enum is ultimately a good idea. I'd be happier with operators where you can just do this:
`myOp.myOutput` and `myOp.myOtherOutput` rather than `myOp.Output[myenum.MY_OUTPUT]` and `myOp.Output[myenum.MY_OTHER_OUTPUT]`, but there is a lot of fixing required before we get to that =/